backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -552,30 +558,33 @@ func (ctx *AppContext) processNewTickets(nt NewTicketsForBlock) {
 	ignoredLowFeeTicketsCount := len(ctx.IgnoredLowFeeTicketsMSA)
 	liveTicketsCount := len(ctx.LiveTicketsMSA)
 	ctx.Unlock()
-	log.Debug("processNewTickets ctx.Unlock")
+	log.Debug("processNewTickets: ctx.Unlock")
 
 	// Log ticket information outside of the handler.
 	go func() {
 		for ticket, msa := range newLiveTickets {
-			log.Infof("added new live ticket %v msa %v", ticket, msa)
+			log.Infof("processNewTickets: added new live ticket %v msa %v", ticket, msa)
```

Comment:
If this is info, the average person might not know what an msa is. Could we write it out? I see you do write it out later.